temporal/api/cloud/cloudservice/v1/request_response.proto

```diff
@@ -217,6 +217,40 @@ message DeleteNamespaceResponse {
     temporal.api.cloud.operation.v1.AsyncOperation async_operation = 1;
 }
 
+message FailoverNamespaceRegionRequest {
+    // The namespace to failover.
+    string namespace = 1;
+    // The id of the region to failover to.
+    // Must be a region that the namespace is currently available in.
+    string region = 2;
+    // Skip graceful failover and only start forceful failover.
```

Comment:
this can essentially result in data-loss / zombies right? Do we need to have a stronger comment here cautioning use of this?

Reply:
This is a good callout. I will remove this one for now and if necessary, we can expose it later.